### ISSUE TITLE
Emacs stops accepting *all* input and says "Unbound suffix"

### ISSUE THREAD

**Issue by waisbrot** [magit/transient#64]
I'm having the same problem as #50 

1. I have a repository (not trivially small; takes maybe 20 seconds to do a full clone)
2. The repository has ~100 branches on `origin` which I fetch
3. I do `b c` and check out from `origin/foo` naming my local branch `foo`
4. I'll want to rebase from master regularly. I do `P C u u` and change it to `origin/master`
5. I want to change my push-default to `origin/foo`, but when I type `p` I get the error

The error says 

> Unbound suffix: '?' (Use 'C-g' to abort, '?' for help)

where `?` is whatever key I type (including `?`). If I type `C-g`, I see no error but I don't escape either. Except that if I type `a` it registers and hilights the `a`s under "Configure branch creation" but then rejects any key after that.

I haven't found a way to get out of this state besides terminating Emacs.

**Comment by tarsius** [comment]
I couldn't reproduce this (obviously using another repository). Please try to narrow this down to what really matters. Does this happen in other repositories? Even if they are much smaller?

Does it only happen if you go through 4 first or does it also happen if you go straight for 5?

**Comment by tarsius** [comment]
Please help me narrow this down by answering my questions.

**Comment by Wagk** [comment]
I have a similar issue where there are times where transient would print a message `Unbound suffix: '$SUFFIX$' (Use 'C-g' to abort, '?' for help)`, and not accept `C-g` inputs until the message disappears from the minibuffer.

I generally get it when git/emacs is lagging/trying to process something (so when the latency I experience is pretty high). This is all anecdotal, however, so I am currently unable to generate a minimal test case.

**Comment by tarsius** [comment]
Make sure you have the latest `transient` installed, just in case. Also it would be useful to know which suffix happens to be unbound.

**Comment by bricewge** [comment]
I have found a way to reproduce this issue with any repository on commit 3bfde52f3a1f274d2d2ddcdfabc9904f4fc5001f with emacs 27.0.91:
1. Open magit status
2. Type `l -Ffoo`: the transient log window appear and the minibuffer contains '--grep=foo'
3. Click away to the magit global status
4. Again type `l -F`, now the minibuffer can't stop screaming '--grep=  [Unbound suffix...' at each key press

I can't exit it with C-g, I'm stuck in transient. 

Related to this there is an other stuck state where where I can navigate between buffers and key presses that aren't bound to the transient buffer can be used but the one bound are bound in all buffer; for example when stuck in log transient, when in a text-mode buffer pressing 'm' will write 'm' while 'l' will open the magit log view. In fact I was trying to reproduce that issue when I stumbled upon a reproducible method for this one.
